modules/cross-account-member/main.tf

```diff
@@ -1,11 +1,16 @@
-provider "aws" {
-  alias = "administrator"
+# Enables/configures Security Hub in administrator account
+module "securityhub_owner" {
```

Comment:
This should not be here. It will be managed separately, in a different call. The cross-account-member module gets invoked for _every_ account. The securityhub owner can only enable security hub in the owner/admin account one time. So doing this here creates a workflow problem. For the test, setup securityhub in the owner account separately, instead.

Reply:
Moved the resource creation to the test.